ui/app/pages/mobile-sync/mobile-sync.component.js

```diff
@@ -145,6 +145,8 @@ export default class MobileSyncPage extends Component {
           this.disconnectWebsockets()
           this.setState({ syncing: false, completed: true })
         }
+
+        return true
```

Comment:
Is this return value used for anything? I glanced through the PubNub docs and couldn't find any mention of it. Maybe the better approach would be to replace the `return false` on line 134 with `return`

Reply:
Good catch, thanks!

(I was going to go back and look at the docs and I forgot to.)